interpretation/deepseismic_interpretation/segyconverter/utils/segyextract.py

```diff
@@ -0,0 +1,343 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+"""
+Methods for processing segy files that do not include well formed geometry. In these cases, segyio
+cannot infer the 3D volume of data from the traces so this module needs to do that manually
+"""
+
+import os
+import math
+import segyio
+import pandas as pd
+import numpy as np
+import json
+
+FAST = "fast"
```

Comment:
Might want to consider adding a comment on sharing how FAST and SLOW are used

Reply:
comment added